.github/CODEOWNERS

```diff
@@ -0,0 +1,11 @@
+# Reviewers to be assigned for any changes in ci/ subdirectory
+ci/*  @quaresmajose @ricardosalveti @sbanerjee-quic
+
+# Reviewers to be assigned for any changes in conf/ subdirectory
+conf/*  @quic-vkraleti @ricardosalveti @sbanerjee-quic
+
+# Reviewers to be assigned for any changes in recipes-kernel/linux/ subdirectory
+recipes-kernel/linux/*  @NainaMehtaQUIC @quic-vkraleti @sbanerjee-quic
+
+# Reviewers responsible for all changes in entire repository 
+*  @quic-vkraleti @ricardosalveti @sbanerjee-quic
```

Comment:
I think the last matching pattern takes the most precedence so this will probably just assign everything to all 3 here.

Reply:
addressed by moving the catchall line to the top.